### ISSUE TITLE
Exception using DataFormatter from POI 4.1

### ISSUE THREAD

**Issue by amergey** [pjfanning/excel-streaming-reader#16]
Code using
org.apache.poi.ss.usermodel.DataFormatter.formatCellValue(Cell, FormulaEvaluator) when Cell is com.github.pjfanning.xlsx.impl.StreamingCell is working with POI 4.0 but is failing with POI 4.1 (java.lang.UnsupportedOperationException)

The code has been changed in POI 4.1 POI (which was not there in POI 4.0) 

```
private boolean isDate1904(Cell cell) {
        if ( cell != null && cell.getSheet().getWorkbook() instanceof Date1904Support) {
            return ((Date1904Support)cell.getSheet().getWorkbook()).isDate1904();

        }
        return false;
    }

```
cell.getSheet().getWorkbook() is throwing the exception (see com.github.pjfanning.xlsx.impl.StreamingSheet.getWorkbook())

**Comment by pjfanning** [comment]
Can you provide a reproducible test case?

**Comment by pjfanning** [comment]
I have published 2.3.2-SNAPSHOT with a possible fix.

**Comment by amergey** [comment]
Here a test that is showing the failure with POI 4.1 (it pass with POI 4.0) 
[test.zip](https://github.com/pjfanning/excel-streaming-reader/files/4374660/test.zip)

**Comment by amergey** [comment]
I will try the 2.3.2 and let you know

**Comment by pjfanning** [comment]
@amergey is that file.xlsx ok to use in tests? Is that data made up or are they real people?

**Comment by amergey** [comment]
they are not real people, but if you need something to include in tests in your repository, let me provide you another file

**Comment by amergey** [comment]
[test.zip](https://github.com/pjfanning/excel-streaming-reader/files/4374926/test.zip)

Here improved version of the test with a simpler excel file you can reuse if you want

**Comment by pjfanning** [comment]
Test and fix released in v2.3.2
